helpers/integration.go

```diff
@@ -200,24 +211,48 @@ type IntegrationPipelineRunOutcome struct {
 	pipelineRunSucceeded bool
 	pipelineRun          *tektonv1.PipelineRun
 	// map: task name to results
-	results map[string]*AppStudioTestResult
+	results map[string]*IntegrationTestTaskResult
 }
 
 // HasPipelineRunSucceeded returns true when pipeline in outcome succeeded
 func (ipro *IntegrationPipelineRunOutcome) HasPipelineRunSucceeded() bool {
 	return ipro.pipelineRunSucceeded
 }
 
+// HasPipelineRunValidTestOutputs returns false when we failed to parse results of TEST_OUTPUT in tasks
+func (ipro *IntegrationPipelineRunOutcome) HasPipelineRunValidTestOutputs() bool {
```

Comment:
I am interested in usecase where we report that parsing of TEST_OUTPUT failed and we are not interested on why it happened.

Reply:
We are not, there is another function that provides errors